### DISCUSSION TITLE
is it possible to use kustomize for creating tenants?

### DISCUSSION THREAD

**Discussion by divramod** [minio/minio#14561] [Q&A]
hey, 

we use kustomize for managing our kubernetes resources.

i wanted to ask, if it is only possible to create tenants via the gui or the kubectl plugin or if it is possible to create a tenant from artifacts via kustomize and if so, where i can find an example?

thx in advance for any hint

**Comment by ravindk89** [comment]
Did you take a look already at our [examples](https://github.com/minio/operator/tree/master/examples/kustomization) in the MinIO Operator repository?
